### DISCUSSION TITLE
Is there a way to enforce strict node ordering (Eg. Ensuring that a document has a title)

### DISCUSSION THREAD

**Discussion by bitboxx** [facebook/lexical#2987] [Q&A] [answered]
Is there a way to configure Lexical follow a strict schema in terms of node ordering? 

For example if we want the document being edited to **always** have...
 - a heading at the top
 - one or more paragraphs (below the heading)
 - references list at the bottom

**Comment by GermanJablo** [comment, marked as answer]
Not at this moment. FYI see this: https://github.com/facebook/lexical/issues/1338

**Comment by nvegater** [comment]
This would be a fantastic feature.
